src/scripts/main.js

```diff
@@ -1 +1,17 @@
 'use strict';
+
+window.addEventListener('hashchange', () => {
+  if (window.location.hash === '#menu') {
+    document.body.classList.add('page__body--with-menu');
+  } else {
+    document.body.classList.remove('page__body--with-menu');
+  }
+});
+
+const form = document.getElementById('myForm');
+
+function handleForm(event) {
+  event.preventDefault();
+}
+
+form.addEventListener('submit', handleForm);
```

Comment:
fields must be cleared after submission

![submit-form](https://user-images.githubusercontent.com/89162624/209441948-94d1526a-adfb-4472-ae30-0ef015f02f15.png)